cypress/pom/footer.js

```diff
@@ -0,0 +1,14 @@
+import { BasePage } from "./base/basePage";
+
+export class PageFooter extends BasePage {
+  constructor() {
+    super('#footer')
+  }
+
+  checkFooterVisible() {
```

Comment:
Don't duplicate code (principle DRY). If two objects have the same behavior, you need to move this behavior to the base class, for example:
1. First of all you need to add get container method inside base page:

> get container() {
        return cy.get(this.CONTAINER, {timeout: 60000});
    }
    
2. Then you need to create general method inside base page what will check container (exist and visible):

> checkContainer(){
        this.container
            .should('exist')
            .should('be.visible');
            return this;
    }
    
At the end, your Base class must looks like:

```
export class BasePage {
    CONTAINER;
    PAGE_NAME;
    PAGE_URL;

    constructor(container, page_name, page_url){
        this.CONTAINER = container;
        this.PAGE_NAME = page_name;
        this.PAGE_URL = page_url;
    }

    get container() {
        return cy.get(this.CONTAINER, {timeout: 60000});
    }

    visit() {
        cy.visit(this.PAGE_URL);
        
        return this
    }

    checkPageUrl() {
        cy.location('href', {timeout: 60000}).should('include', this.PAGE_URL);

        return this
    }

    checkContainer(){
        this.container
            .should('exist')
            .should('be.visible');

            return this;
    }

}